week2/teacher-live-coding/src/backend/middleware.js

```diff
@@ -2,7 +2,6 @@
 // Take a look inside the folder!
 const express = require("express");
 const app = express();
-const miniRouter = require("./mini-router");
```

Comment:
`miniRouter` isn't used here. Not sure if it is supposed to be eventually used here in the live coding?